src/views/swap/swapActions.js

```diff
@@ -15,3 +20,56 @@ export const setSwapAmount = (sent, received) => ({
     received,
   },
 });
+
+/**
+ * @param invoice swap invoice
+ */
+export const setSwapInvoice = invoice => ({
+  type: actionTypes.SET_SWAP_INVOICE,
+  payload: invoice,
+});
+
+/**
+ * @param {boolean} success
+ * @param {object} response
+ */
+export const swapResponse = (success, response) => ({
+  type: actionTypes.SWAP_RESPONSE,
+  payload: {
+    success,
+    response,
+  },
+});
+
+export const swapRequest = () => ({
+  type: actionTypes.SWAP_REQUEST,
+});
+
+export const startSwap = (swapInfo, cb) => {
+  const url = 'http://localhost:9001/createswap';
+  return dispatch => {
+    dispatch(swapRequest());
+    axios
+      .post(url, {
+        pairId: 'LTC/BTC',
```

Comment:
We need a LTC and T-LTC part, Its fine not having the logos for this version since the BTC logos are not included here as well. @michael1011